content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,132 @@
+* Isolating terminal and input-related functionalities from the rest of KDBG ([#5188](https://github.com/reactos/reactos/pull/5188));
+* Fixed an input buffer reading bug in `KdbpReadCommand()` and `KdReceivePacket()` routines ([#4914](https://github.com/reactos/reactos/pull/4914));
+* Reworked the boot-phase initialization of KD and KDBG ([#4892](https://github.com/reactos/reactos/pull/4892));
+* Moved the code handling of Dmesg buffer from screen provider to KDBG provider ([#5143](https://github.com/reactos/reactos/pull/5143));
+
+The goal of this work is to have a terminal version of KDCOM and separate KDBG into a dedicated KDCOM-like extension DLL, such as `kdrosdbg.dll` (see [#4600](https://github.com/reactos/reactos/pull/4600)).
+All of this effort is further coordinated with Timo Kreuzer.
+
+## UEFI Boot
+Thanks to the work of Timo Kreuzer (tkreuzer) our AMD64 build is now capable of booting on an increasing number of hardware.
+Thanks to Timo's work, we are working towards supporting modern devices with UEFI class 3 firmware.
+Work has been underway since the beginning of the year to transition FreeLoader, our default bootloader for ReactOS, to support UEFI on ARM32, ARM64, x86 and of course: AMD64.
+Hermès has been developing a system for passing the UEFI framebuffer information in a fashion that allows Windows XP to run on UEFI systems, while Justin Miller (The_DarkFire_) has been developing the UEFI freeloader build.
+While Justin Miller (The_DarkFire_) has been developing the UEFI freeloader build.
+
+On top of supporting booting ReactOS, other features are being built such as EFI chainloading and a bootmgfw compatible build of FreeLoader. These features would add boot management capabilities and allow modern Windows systems to bootstrap our favorite bootloader.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/uefi.png" src="/img/blogs/newsletter-103/uefi.png" caption="Running ReactOS on a Steam Deck" >}}
+{{< figure link="/img/blogs/newsletter-103/htc_hd2.png" src="/img/blogs/newsletter-103/htc_hd2.png" caption="Running FreeLoader on a HTC HD2" >}}
+{{< figure link="/img/blogs/newsletter-103/lumia950xl.png" src="/img/blogs/newsletter-103/lumia950xl.png" caption="Running FreeLoader on a Lumia 950XL" >}}
+{{< figure link="/img/blogs/newsletter-103/bootmgfw_freeldr.png" src="/img/blogs/newsletter-103/bootmgfw_freeldr.png" caption="ReactOS FreeLoader boot entry shown in Windows Boot Manager in a Framework laptop" >}}
+{{< figure link="/img/blogs/newsletter-103/apple.png" src="/img/blogs/newsletter-103/apple.png" caption="Running FreeLoader on a Apple iPhone 5c" >}}
+{{< /gallery >}}
+
+It's worth noting that FreeLoader runs natively on the aforementioned phones, without emulators of any kind.
+
+**WARNING:** UEFI support in ReactOS is experimental, under heavy development and is not yet ready for the general use. We currently cannot give an estimate as to when it will be done, so please stay tuned!
```

Comment:
```suggestion
**WARNING:** UEFI support in ReactOS is experimental, under heavy development and is not yet ready for the general use.
We currently cannot give an estimate as to when it will be done, so please stay tuned!
```